src/user_config.yaml

```diff
@@ -56,7 +56,7 @@ hdfs_paths:
   feather_path: "staging/feather"
   # Freezing data paths
   frozen_data_staged_output_path: "02_freezing/frozen_data_staged" # path for folder
-  frozen_data_staged_path: "02_freezing/frozen_data_staged/2023_FROZEN_staged_BERD_full_responses_24-08-07_v906" # path to file
+  frozen_data_staged_path: "02_freezing/frozen_data_staged/2023_FROZEN_staged_BERD_full_responses_24-08-07_v906" # pragma: allowlist secret
```

Comment:
I had to add the pragma: allowlist secret as this line had falsely been flagged as possibly containing a secret in the pre-commit.